sdk/core/src/http_client/reqwest.rs

```diff
@@ -24,6 +26,10 @@ impl HttpClient for ::reqwest::Client {
 
         let reqwest_request = match body {
             Body::Bytes(bytes) => req.body(bytes).build(),
+            // Hack... Temporarily remove for wasm32
+            // Need to implement properly when reqwest has support for streaming body.
+            // https://github.com/seanmonstar/reqwest/pull/1576
```

Comment:
https://github.com/seanmonstar/reqwest/pull/1576 looks good to me, but has not been merged yet.

Reply:
Looks like this function was merged to reqwest main last week via https://github.com/seanmonstar/reqwest/pull/1713.

Now just need to wait for it to appear in a release...

Reply:
Ok, this is available in reqwest 0.11.14:
https://github.com/seanmonstar/reqwest/releases/tag/v0.11.14

I'll try updating the PR to use the new function...